app/Models/Context.php

```diff
@@ -162,7 +164,7 @@ public static function updateUsingRequest() {
 		self::$state = Minz_Request::param(
 			'state', self::$user_conf->default_state
 		);
-		$state_forced_by_user = Minz_Request::param('state', false) !== false;
```

Comment:
This is also an example of minor personal preference that I would rather avoid changing, but it does not matter for this time